gpt_review/gpt_review.py

```diff
@@ -86,7 +86,7 @@ def review():
                 f"\n diff: {diff}",
             },
         ],
-        model=AZURE_DEPLOY_MODEl,
+        model=AZURE_DEPLOY_MODEL,
         functions=functions,
```

Comment:
*メンテナンス性* 観点の *high* レベルの指摘
 このコードでは'AZURE_DEPLOY_MODEL'環境変数を何度も参照しています。タイプミスを防ぐため、このような定数は一度だけ定義して参照するとよいでしょう。

```suggestion
'AZURE_DEPLOY_MODEL = os.getenv("AZURE_DEPLOY_MODEL")'というコードを一度書けば、後は'AZURE_DEPLOY_MODEL'と書くだけで参照できます。
```